doc/js_tutorials/js_assets/js_dnn_example_helper.js

```diff
@@ -0,0 +1,98 @@
+getBlobFromImage = function(inputSize, mean, std, swapRB, image) {
+    let mat;
+    if(typeof(image) === 'string') {
+        mat = cv.imread(image);
+    } else {
+        mat = image;
+    }
+
+    let matC3 = new cv.Mat(mat.matSize[0], mat.matSize[1], cv.CV_8UC3);
+    cv.cvtColor(mat, matC3, cv.COLOR_RGBA2RGB);
```

Comment:
Why 2RGB? C++ and Python users work with BGR images (from `imread`) so by default they expect this layout and `swapRB` means that network process RGB. Current logic is reversed and may confuse so I'd like to recommend to keep `swapRB=true` for RGB and `swapRB=false` for BGR. So 
```
cv.cvtColor(mat, matC3, cv.COLOR_RGBA2BGR);
```
first

Reply:
@akineeic , any response?

Reply:
This sounds more suitable. I did that in the latest commit.